### PR TITLE
Setup/express.js

### PR DESCRIPTION
**Pull request by mrfong**

### CODE REVIEW

**Comment by tawfiknasser** [comment]
Can you describe your changes ?
why did you changed the files names ?

**Comment by mrfong** [comment]
> Can you describe your changes ?
> why did you change the files names ?



> Can you describe your changes ?
> why did you change the files names ?

"delete unused files, made a new files for the functions we're going to be using for the routing points"

File: server/app.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 const express = require('express');
-
+const compression = require('compression');
+const bodyParser = require('body-parser');
 const controllers = require('./controllers/routes');
 
 const app = express();
 
+app.disable('x-powered-by');
+app.use(bodyParser.json());
```
Comment (tawfiknasser) [inline server/app.js]: "you don't need to use body parser. Instead, you can use the built-in express functionality like this:
`app.use(express.json());` "

File: webpack.config.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-module.exports = { entry: './client/src/index.js',
+module.exports = {
+  entry: './client/src/index.js',
   output: { path: path.join(__dirname, '/dist'), filename: 'bundle.js'},
   module: { rules: [{ test: /\.(js|jsx)$/,
     exclude: /node_modules/,
-    use: { loader: 'babel-loader'}}]},
-  plugins: [ new HtmlWebpackPlugin({ template: './client/src/index.html'})]};
+    use: { loader: 'babel-loader' }} ]},
+  plugins: [ new HtmlWebpackPlugin({ template: './client/src/index.html' }) ]};
```
Comment (tawfiknasser) [inline webpack.config.js]: please add the ESLint auto fix

File: server/controllers/routes.js [hunk 3]
```diff
@@ -7,5 +7,54 @@ routers.get('/', (req, res) => {
...
+  res.redirect('/parent-profile');
+});
+
+routers.post('/create-user', (req, res) => {
+  // check the database if user exists
+
+  // create new entry in database with parent's info
+
+  // direct them to /parent-sign-in, and direct to /parent-profile
+  res.redirect('/parent-profile');
```
Comment (tawfiknasser) [inline server/controllers/routes.js]: use React router instead, in the front end to navigate between routes.
This redirect will cause you conflict's when you serve the built public file

File: webpack.config.js (hunk 2 above)
Comment (tawfiknasser) [reply webpack.config.js]: later

**Pull request merged**